docs/src/notation.md

```diff
@@ -0,0 +1,24 @@
+# Notation overview
+
+Since manifolds include a reasonable amount of elements and functions, the
+following list tries to keep an overview of used notation throughout `Manifolds.jl`.
+The order by terminology. Thy might be used in a plain form within the code or
+when referring to that code. This is for example the case the caligraphic symbols.
+
+Within the documented functions the utf8 symbols are used whenever possible,
+as long as that renders still in TeX within this documentation.
+
+A
+
+| Symbol | Description | Also used | Comment |
+|:--:|:-------------- |:--:|:--- |
+| ``T^*_x ℳ`` | The cotangent space at ``x`` | | |
```

Comment:
Either way, doesn't render correctly for some reason: see https://julianlsolvers.github.io/Manifolds.jl/previews/PR104/notation.html

Reply:
That table took me quite some time, neither `$`, _nor_ ` `` ` seem to render correctly within tables :/

Reply:
Hmm that's weird. I don't have time to troubleshoot this right now, but I once got math working successfully in tables, but maybe it was a fluke. See https://raw.githubusercontent.com/arviz-devs/ArviZ.jl/master/docs/src/api.md. If that doesn't help, then it might help to check slack or open a Documenter.jl issue.

Reply:
I have not yet checked (exercise in 10 minutes, yes that's around 7:20 am 😴 ) but it might be that when there is no space between `|` and ` `` ` some rendering gets confused or something. Will investigate during the day (hopefully).